docs/guide/idp-app-token.md

```diff
@@ -0,0 +1,69 @@
+# Fake IdP SAML App Canarytoken
+
+## What is an IdP SAML2 App Canarytoken
+
+This Canarytoken works by being installed on your Identity Provider SSO dashboard disguised as a normal app. When the app is opened, the Canarytoken sends an alert with the identity of the user whose dashboard it was used from. It can optionally redirect to a specified URL.
```

Comment:
> Detect identity compromise with the IdP App Canarytoken. It works simply: you setup a juicy sounding app in your SSO dashboard, and if the app is ever opened then you get an email with the identity of the user who clicked on the fake app. This lets you identify exactly which user was compromised.

> Optionally it can redirect the user to a URL of your choice, after they've opened the app. This helps maintain the ruse for longer.